base/essentials.jl

```diff
@@ -712,6 +712,33 @@ function invokelatest(@nospecialize(f), @nospecialize args...; kwargs...)
     Core._apply_latest(inner)
 end
 
+"""
+   invokefrozen(f, rt, args...; kwargs...)
+
+Calls `f(args...;kwargs...)` but allows the method of `f` at its current age to
+be directly executed without a world-age error. This is useful in specialized
+circumstances, such as code generation, where post-265 automatic function
+updating behavior may be non-sensical and impede usage. This has a very
+small overhead compared to calling `f` directly, and is much faster than
+`invokelatest`.
+"""
+@inline function invokefrozen(f, rt, args...; kwargs...)
+    if isempty(kwargs)
+        return _invokefrozen(f, rt, args)
+    end
+    # We use a closure (`inner`) to handle kwargs.
+    inner() = f(rt, args...; kwargs...)
+    _invokefrozen(inner)
```

Comment:
There's no `_invokefrozen(f)` method, I think this needs to be `_invokefrozen(inner,rt)`